<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiSell</class>
 <widget class="QWidget" name="GuiSell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>20</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="accountInfo">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="stockNameName">
     <property name="text">
      <string>股票名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="stockName">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="stockCodeName">
     <property name="text">
      <string>股票代码</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="stockCode">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="placeholderText">
      <string>6位数字</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sellPriceName">
     <property name="text">
      <string>卖出价格</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="SellDoubleSpinBox" name="sellPrice">
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sellableQtyName">
     <property name="text">
      <string>可卖股数</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sellQtyName">
     <property name="text">
      <string>卖出数量</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="sellBtn">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>卖出</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="SellSpinBox" name="sellQty">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="sellableQty">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SellDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui_sell.h</header>
  </customwidget>
  <customwidget>
   <class>SellSpinBox</class>
   <extends>QSpinBox</extends>
   <header>gui_sell.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stockCode</tabstop>
  <tabstop>sellPrice</tabstop>
  <tabstop>sellQty</tabstop>
  <tabstop>sellBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>stockCode</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GuiSell</receiver>
   <slot>UserEditStockCode(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellPrice</sender>
   <signal>valueChanged(double)</signal>
   <receiver>GuiSell</receiver>
   <slot>UserEditSellPrice(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellQty</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GuiSell</receiver>
   <slot>UserEditSellQty(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UserEditStockCode(QString)</slot>
  <slot>UserEditSellPrice(double)</slot>
  <slot>UserEditSellQty(int)</slot>
 </slots>
</ui>
